src/snowcli/cli/snowpark/package/manager.py

```diff
@@ -61,8 +62,7 @@ def upload(file: Path, stage: str, overwrite: bool):
 
 def create(name: str):
```

Comment:
```suggestion
def create(archive_name: str):
```
WDYT? Or something more meaningful

Reply:
`zip_name`? To be consistent

Reply:
SGTM!